<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightScreen</class>
 <widget class="QWidget" name="FightScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="TextImage" name="char_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets.qrc">:/assets/images/test.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="TextImage" name="enemy">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets.qrc">:/assets/images/test.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ProgressBar" name="char_hp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%p/100</string>
   </property>
  </widget>
  <widget class="ProgressBar" name="enemy_hp">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%p/100</string>
   </property>
  </widget>
  <widget class="TextImage" name="char_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextImage</class>
   <extends>QLabel</extends>
   <header>data/widgets/textimage.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>data/widgets/progressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
